### PR TITLE
docs: recommend noble in Docker images

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: docs/src/docker.md [hunk 1]
```diff
@@ -108,8 +108,6 @@ See our [Continuous Integration guides](./ci.md) for sample configs.
 See [all available image tags].
 
 We currently publish images with the following tags:
-- `:next` - tip-of-tree image version based on Ubuntu 22.04 LTS (Jammy Jellyfish).
```
Comment (mxschmitt) [inline docs/src/docker.md:111]: This was a false statement, since we stopped publishing ToT Docker images.

**Comment by mxschmitt** [comment]
lets wait one more release as per offline discussion.

File: utils/docker/publish_docker.sh [hunk 2]
```diff
@@ -28,36 +23,20 @@ fi
...
 
-if [[ "$RELEASE_CHANNEL" == "stable" ]]; then
-  FOCAL_TAGS+=("focal")
-fi
-
 # Ubuntu 22.04
 JAMMY_TAGS=(
-  "next-jammy"
   "v${PW_VERSION}-jammy"
   "v${PW_VERSION}"
```
Comment (dgozman) [inline utils/docker/publish_docker.sh:32]: So, our default tag is still jammy? Why recommend noble?

Comment (mxschmitt) [reply utils/docker/publish_docker.sh]: Done.

**Review by dgozman — APPROVED** [review]

File: docs/src/docker.md [hunk 3]
```diff
@@ -108,8 +108,6 @@ See our [Continuous Integration guides](./ci.md) for sample configs.
...
-- `:next-jammy` - tip-of-tree image version based on Ubuntu 22.04 LTS (Jammy Jellyfish).
 - `:v%%VERSION%%` - Playwright v%%VERSION%% release docker image based on Ubuntu 22.04 LTS (Jammy Jellyfish).
```
Comment (mxschmitt) [inline docs/src/docker.md:111]:
```suggestion
- `:v%%VERSION%%` - Playwright v%%VERSION%% release docker image based on Ubuntu 24.04 LTS (Noble Numbat).
```

**Pull request merged**